concordium-std/src/test_infrastructure.rs

```diff
@@ -1645,6 +1645,12 @@ impl<State: Serial + DeserialWithState<TestStateApi>> TestHost<State> {
         *self.contract_balance.borrow_mut() = amount;
     }
 
+    /// Set the contract address.
+    /// This is used to redirect queryies for the balance of the contract
```

Comment:
```suggestion
    /// This is used to redirect queries for the balance of the contract
```

Reply:
`This is used to redirect..` makes it sound like that is the primary purpose of the method, which it isn't. 

Perhaps something like:
```
Set the contract address.

This will be the address returned by `ctx.self_address`.

This is also used to redirect queries...
```

Reply:
Ah, I guess I thought I was looking at the one from context.
Since it is a very rare use case, then this is probably fine then.